### ISSUE TITLE
False positive for Proj3002

### ISSUE THREAD

**Issue by Frulfump** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#760]
I get a detection for

```xml
...
  <ItemGroup>
    <!-- Benchmarks -->
    <PackageVersion Include="BenchmarkDotNet" Version="0.15.8" />
...
```
But there's no code being commented out, just a helpful comment for me

https://dotnet-project-file-analyzers.github.io/rules/Proj3002.html

**Comment by Corniel** [comment]
Note that what I would do is:

``` xml
 <ItemGroup Label="Benchmarks">
    <PackageVersion Include="BenchmarkDotNet" Version="0.15.8" />
```

How the comments checkers works is by checking if the `XComment` text [could be an XML tag](https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/blob/main/src/DotNetProjectFile.Analyzers/Analyzers/Helpers/CommentChecker.cs). But for single words the `<{text.Trim()}>` match check is too easy triggering this rule.

**Comment by Corniel** [comment]
@Frulfump Would you please be so kind to review #762?

**Comment by Frulfump** [comment]
Confirm fixed in 1.12.0
